server/game/core/event/EventWindow.js

```diff
@@ -57,6 +59,12 @@ class EventWindow extends BaseStepWithPipeline {
     setCurrentEventWindow() {
         this.previousEventWindow = this.game.currentEventWindow;
         this.game.currentEventWindow = this;
+        if (this.resolveTriggersAfter) {
+            this.triggeredAbilityWindow = new TriggeredAbilityWindow(this.game, this, AbilityType.Triggered);
```

Comment:
Since we're initializing `this.triggeredAbilityWindow` here, can remove the initialize-if-null logic in `triggerEventsForWindow` (and probably just omit that method completely tbh)

Reply:
Done. Some EventWindows still won't have a TriggeredAbiltyWindow at all(see the other thread about setCurrentEventWindow) so I use this.triggeredAbilityWindow?. in a few spots.